### PR TITLE
LPS-88520 Add functional test

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test

File: modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+	@Test
+	public void testBlogPostsLinkExistsInContentSpace() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).when(
+		).get(
+			_rootEndpointURL.toExternalForm()
```
Comment (cgoncas) [inline modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java]: We can use contentSpaceHref here

File: modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+			"_embedded.BlogPosting[0].keywords",
+			IsEqual.equalTo(Arrays.asList("keyword1", "keyword2", "keyword3"))
+		);
+
+		_deleteBlog(blogHref);
+	}
+
+	@Test
+	public void testUpdateBlog() throws Exception {
+		String blogHref = ApioClientBuilder.given(
```
Comment (cgoncas) [inline modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java]: could be fine to move this to a new method as deleteBlog?

File: modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+		).when(
+		).post(
+			_blogPostsHref
+		).then(
+		).extract(
+		).path(
+			"_links.self.href"
+		);
+
+		_deleteBlog(blogHref);
```
Comment (cgoncas) [inline modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java]: It is in general not a good practice to do this with the _deleteBlog, when this test fails, all the code that you have here is  the one to obtain the blogHref. Looks like the _deleteBlog code is just for cleaning, no the test itself

**Comment by victorg1991** [comment]
ci:close

**Pull request closed without merging**